packages/plugin-ext/src/hosted/node/scanners/scanner-theia.ts

```diff
@@ -384,12 +384,13 @@ export class TheiaPluginScanner implements PluginScanner {
     }
 
     private readTaskDefinition(pluginName: string, definitionContribution: PluginTaskDefinitionContribution): TaskDefinition {
+        const propertyKeys = definitionContribution.properties ? Object.keys(definitionContribution.properties) : [];
```

Comment:
(out-of-PR-scope) @theia-ide/task-extension why do we care only about keys? what about values?